src/app/child-dev-project/children/child-details/grouped-child-attendance/grouped-child-attendance.component.html

```diff
@@ -10,4 +10,32 @@
     >
     </app-activity-attendance-section>
   </mat-tab>
+
+  <mat-tab *ngIf="archiveActivities.length > 0" label="Previous Activities">
+    <p i18n>
+      This section displays historical activities in which the student was
+      previously enrolled.
+    </p>
+
+    <mat-form-field>
+      <mat-label>Select a Previous Activity</mat-label>
+      <mat-select (selectionChange)="onActivityChange($event.value)">
+        <mat-option
+          *ngFor="let archiveActivity of archiveActivities"
+          [value]="archiveActivity.title"
```

Comment:
probably better if you set the object as value directly (ˋ[value]="archiveActivity"ˋ). Then you could also simply do ˋ(selectionChange)="selectedArchivedActivity=$event.value"ˋ above (and replace the array of selected activities + boolean selectedActivity with simply one variable storing the currently selected one)